odbc/src/api/definitions.rs

```diff
@@ -13,10 +13,12 @@ macro_rules! map {
 
 #[derive(Copy, Clone, Debug, PartialEq, FromPrimitive)]
 pub enum SqlBool {
-    SqlFalse = 0,
-    SqlTrue,
+    False = 0,
+    True,
```

Comment:
Yep, I think these are better names. I know @rychipman always likes to avoid "stutter" in type names, too :)